src/views/homepage/components/learn-section/learn-section.module.css

```diff
@@ -0,0 +1,81 @@
+.learnSection {
+  --container-max-width: 1240px;
+  --padding-y: 64px;
+  --padding-x: 24px;
+
```

Comment:
Polish task to dial in these sizes across components https://app.asana.com/0/1202111817660946/1202127264881983